src/table/utils/__tests__/handle-key-press.spec.ts

```diff
@@ -274,6 +274,17 @@ describe('handle-key-press', () => {
       expect(nextCol).toBe(0);
     });
 
+    it('should go one cell forward when the last cell is not selected and the key is pressed down to bottom Totals row', () => {
```

Comment:
This description does not make sense. And I don't see how this test is needed. What is new in this PR is that you don't go to the last row if allowedRows.bottom === 1. That could use a test